Prevent puppet's build script from gzipping the manpages.  Our version
of man doesn't understand manpages with a .gz extension

--- puppet-3.2.2/install.rb.orig	2013-01-04 14:05:09.892870020 -0700
+++ puppet-3.2.2/install.rb	2013-01-04 14:05:40.466062888 -0700
@@ -146,9 +146,6 @@
       FileUtils.chmod(0755, om)
       FileUtils.install(mf, omf, {:mode => 0644, :verbose => true})
     end
-    gzip = %x{which gzip}
-    gzip.chomp!
-    %x{#{gzip} -f #{omf}}
   end
 end
 
